practice-areas/design-and-research/README.md

```diff
@@ -1,41 +1,102 @@
 ---
-title: Our design team principles
+title: Design practice area
 ---
 
-# Our design team principles
+# Design practice area
 
-## We start with discovery, not solutions
+The CivicActions design team works with our clients on projects and partners with our other [internal practice areas](https://guidebook.civicactions.com/en/latest/practice-areas/). Our design area includes content designers, product designers, design educators, and service designers. We use the **#just-design** slack channel for our communications and the **#design** slack channel to communicate across other practice areas. We have regular All Creatives Calls (ACC) and post bi-monthly slack announcements.
```

Comment:
@kellysmith1008 could we make the link for 'internal practice areas' relative? Here is an example of the same link on the Drupal page https://github.com/CivicActions/guidebook/blob/master/practice-areas/engineering/drupal/README.md?plain=1#L7. I believe for this page the link would be:

[internal practice areas](../../practice-areas/README.md)

Reply:
@dmundra We sure can! Well, you can, I need to learn how! Are you saying to replace the guidebook links with ../../../{page-name}/README.md links instead? Sorry if that's a silly question, but want to make sure I understand before I change it (if I can figure out how to change it!)

Reply:
> Are you saying to replace the guidebook links with ../../../{page-name}/README.md links instead?

Yup exactly. I fixed my first comment to show the code snippet correctly. Sorry about that.

> Sorry if that's a silly question, but want to make sure I understand before I change it (if I can figure out how to change it!)

Not a silly question.

Reply:
@dmundra Ok, I think I made both changes, but please let me know if I did it incorrectly!

Reply:
The changes look great. I will comment on the one gotcha for the professional development link.